sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/Config/ServiceBusHostBuilderExtensionsTests.cs

```diff
@@ -134,7 +130,7 @@ public void AddServiceBus_ThrowsArgumentNull_WhenServiceBusOptionsIsNull()
 
             var exception = Assert.Throws<ArgumentNullException>(() => host.Services.GetServices<IExtensionConfigProvider>());
 
-            Assert.AreEqual("serviceBusOptions", exception.ParamName);
+            Assert.AreEqual("options", exception.ParamName);
```

Comment:
Could this be a `nameof` to avoid the magic string?

Reply:
Actually, doesn't look like this is exposed.